src/gate/parserGnet/parserGnet.cpp

```diff
@@ -0,0 +1,307 @@
+/**
+ * \brief Implements translation from Liberty format file to GNet.
+ * \author <a href="mailto:anushakov@edu.hse.ru">Aleksander Ushakov</a>
+ */
+#include "parserGnet.h"
+#include "base/model/signal.h"
+#include "gate/model/gate.h"
+#include "gate/model/gnet.h"
+#include "gate/simulator/simulator.h"
+#include "passes/techmap/libparse.h"
+#include <kernel/celltypes.h>
+#include <kernel/yosys.h>
+#include <iostream>
+#include <map>
+#include <string>
+#include <vector>
+
+namespace gmodel = eda::gate::model;
+namespace lib = Yosys::hashlib;
+namespace yosys = Yosys::RTLIL;
+
+using gate = gmodel::Gate;
+using gsymbol = gmodel::GateSymbol;
+using gid = gmodel::Gate::Id;
```

Comment:
Type names should start with a Capital letter, consider preserving the names:
```suggestion
using gmodel::Gate;
using gmodel::GateSymbol;
using GateId = gmodel::Gate::Id;
```